packages/website/containers/Layout.tsx

```diff
@@ -1,27 +1,46 @@
 // @ts-nocheck
 
 import { useNearHooksContainer } from "@cura/hooks";
-import { Container, Box, Spinner } from "theme-ui";
+import { Container, Box, Spinner, Text } from "theme-ui";
 import { useRecoilValue } from 'recoil'
 
 import Header from "./Header";
 import { indexLoaderState } from '../state/recoil'
 
 import Menu from "./Menu";
 
+
 export default function Layout({ children, requireAuth = false, page }) {
 
   const { accountId } = useNearHooksContainer();
    
   const indexLoader = useRecoilValue(indexLoaderState)
 
   return (
-    <>
+    <Box
+    	sx={{
+    		height: '100vh',
+    		display: 'flex',
+    		flexDirection:'column'
+    	}}
+    >
       <Header />
       <Menu accountId={accountId} />
-      <Container>
-      	{indexLoader ? (
-            <Box
+      <Container
+      	sx={{
+      		height: '100%',
+      		justifyContent: (requireAuth && !accountId) && 'center',
+            display: (requireAuth && !accountId) && 'flex',
+            alignItems: (requireAuth && !accountId) && 'center',
```

Comment:
I think it would be better if we just separated the `Please connect` Box  from Container, so we don't have this many checks in style. Other than that, everything seems great

Reply:
Yeah. Agree with that.